Sources/Plasma/FeatureLib/pfAnimation/plViewFaceModifier.cpp

```diff
@@ -55,10 +55,8 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 #include "plAvatar/plArmatureMod.h"
 
 plViewFaceModifier::plViewFaceModifier()
-:   fFacePoint(0,0,0),
-    fLastDirY(0,1.f,0),
-    fScale(1.f,1.f,1.f),
-    fOffset(0,0,0)
+    : fFacePoint(0,0,0), fLastDirY(0,1.f,0), fScale(1.f,1.f,1.f), fOffset(0,0,0),
```

Comment:
Since you've added default constructors to hsScalarTriple to zero-initialize all fields, any initializers that set to (0, 0, 0) can now be omitted...